fdbcli/TenantCommands.actor.cpp

```diff
@@ -293,9 +293,9 @@ ACTOR Future<bool> tenantDeleteIdCommand(Reference<IDatabase> db, std::vector<St
 				fmt::print(stderr, "ERROR: delete by ID should only be run on a management cluster.\n");
 				return false;
 			}
-			if (sscanf(tokens[2].toString().c_str(), "%lld%n", &tenantId, &n) != 1 || n != tokens[2].size() ||
+			if (sscanf(tokens[2].toString().c_str(), "%ld%n", &tenantId, &n) != 1 || n != tokens[2].size() ||
```

Comment:
I guess probably this should use the macro for the integer type:

```suggestion
			if (sscanf(tokens[2].toString().c_str(), "%" PRId64 "%n", &tenantId, &n) != 1 || n != tokens[2].size() ||
```